app/javascript/shared/components/ui/MutiselectDropdown.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <div v-on-clickaway="onCloseDropdown" class="dropdown-wrap">
+    <button :v-model="value" class="button-input" @click="toggleDropdown">
+      <Thumbnail
+        v-if="value && value.name && value && value.id"
+        :src="value && value.thumbnail"
```

Comment:
Use `value.thumbnail` directly since you are already checking the value is exist or not using `v-if`